networkx/algorithms/components/biconnected.py

```diff
@@ -417,7 +417,11 @@ def articulation_points(G):
            Communications of the ACM 16: 372–378. doi:10.1145/362248.362272
 
     """
-    return _biconnected_dfs(G, components=False)
+    seen = set()
+    for articulation in _biconnected_dfs(G, components=False):
+        if articulation not in seen:
+            seen.add(articulation)
+            yield articulation
```

Comment:
Can you change the documentation from

```rest
     Returns
     -------
     articulation points : generator
         generator of nodes
```
to
```
     Yields
     ------
     node
         An articulation point in the graph.
```
since this function is now an iterator generator instead of a function that returns a generator.